chat/chat-tool/tool/src/webapp/js/chatscript.js

```diff
@@ -156,12 +156,14 @@ var chatscript = {
 	deleteMessage : function(messageId) {
 		var me = this;
 		var removeModal = $("#removemodal");
-		removeModal.modal("hide");
+		var modal = bootstrap.Modal.getInstance(removeModal);
```

Comment:
Does this definitely work? I thought BS-5 modal's needed a dom element? removeModal is a jquery object, isn't it?

Reply:
It works, I have tested it multiple times. and it only works if I use getInstance(). And you are right b-5 modal will only work with a dom element. It has a dom element. 158

Reply:
From https://api.jquery.com/id-selector

"Calling jQuery() (or $()) with an id selector as its argument will return a jQuery object containing a collection of either zero or one DOM element."

I'm just wondering. Maybe getInstance takes a jquery collection, too.